WPMedia/fusion-news-theme-blocks
# updated large promo for resizer

## Description

Update Large Promo to use image resizer

## Jira Ticket

- [TMEDIA-939](https://arcpublishing.atlassian.net/browse/TMEDIA-TMEDIA-939)

## Acceptance Criteria

- Large Promo block is using image resizer v2

## Test Steps

1. Checkout this branch `git checkout TMEDIA-939-large-promo-resizer`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/large-promo-block`
3. Add the large promo block to a page and configure the content source _id - CTAR7XCGLBCAXCHLJ6D5CZEGUA
4. Verify the Image is now using resizer v2 to display image
5. Test the integrated photo search to verify images selected from the integrated photo search also using resizer v2

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
